CRM/Civicase/Helper/XmlProcessor.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+/**
+ * Wrapper class for xml settings.
+ */
+class CRM_Civicase_Helper_XmlProcessor {
```

Comment:
I think the name of this helper file should be something like `CaseSetting` Helper. It really has nothing to do with XML processors, the XML processor class is only being used to help fetch the settings. 

If we name it a `CaseSetting`. I think we should also rename the `get` function to 	`getDefaultValue` because what it returns in essence is the default value for a case setting.